mesonbuild/compilers/mixins/visualstudio.py

```diff
@@ -356,10 +356,10 @@ def get_crt_compile_args(self, crt_val: str, buildtype: str) -> T.List[str]:
         crt_val = self.get_crt_val(crt_val, buildtype)
         return self.crt_args[crt_val]
 
-    def has_func_attribute(self, name: str, env: 'Environment') -> T.Tuple[bool, bool]:
+    def has_func_attribute(self, name: str, env: 'Environment') -> CompileCheckResult:
         # MSVC doesn't have __attribute__ like Clang and GCC do, so just return
         # false without compiling anything
-        return name in {'dllimport', 'dllexport'}, False
+        return CompileCheckResult(name in {'dllimport', 'dllexport'}, False)
```

Comment:
I would either remove the `False`, or use keyword `cached=False`, otherwise we have to know what is the meaning of that `False`...

Reply:
We don't even need the false here, since the default value is false.